lib/gigya_flutter_plugin.dart

```diff
@@ -154,6 +155,15 @@ class GigyaSdk with DataMixin {
         return _timeoutError();
       });
 
+  /// Forgot password using LoginId
+  Future<void> forgotPassword(loginId) =>
```

Comment:
Should be Future<Map<String, dynamic>>. 
We want to provide a response even if the flow is not resolvable.

Reply:
Done